pkg/controller/swift/swift_controller.go

```diff
@@ -155,6 +155,10 @@ func (r *ReconcileSwift) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcile.Result{}, err
 	}
 
+	if err = contrail.CreateAccount("swift", request.Namespace, r.client, r.scheme, swift); err != nil {
```

Comment:
should we keep track on this account or not? I mean if we watch on this we will be sure that it won't disappear

Reply:
This is the same approach as in other controllers. I don't want to change that. In my opinion this should be unified to a one account and managed by manager.